Library/Homebrew/test/os/mac/diagnostic_spec.rb

```diff
@@ -39,4 +39,22 @@
     expect(checks.check_ruby_version)
       .to match "Ruby version 1.8.6 is unsupported on 10.12"
   end
+
+  specify "#check_broken_sdks" do
+    allow(OS::Mac).to receive(:sdk_locator).and_return(OS::Mac::CLT.sdk_locator)
+
+    allow_any_instance_of(OS::Mac::CLTSDKLocator).to receive(:all_sdks)
+      .and_return([
+                    OS::Mac::SDK.new(OS::Mac::Version.new("11"), "/some/path/MacOSX.sdk", :clt),
+                    OS::Mac::SDK.new(OS::Mac::Version.new("10.15"), "/some/path/MacOSX10.15.sdk", :clt),
+                  ])
```

Comment:
Side note: am I the only one who dislikes this indentation and prefers [`consistent` over `special_inside_parentheses`](https://www.rubydoc.info/gems/rubocop/1.9.1/RuboCop/Cop/Layout/FirstArrayElementIndentation)?

Reply:
Yeh, consistent is much nicer. Can you submit a PR to change the rule and autocorrect?

Reply:
Yes, I'll do that separately (maybe after this is merged).